docling_core/types/doc/document.py

```diff
@@ -300,7 +300,7 @@ class TableCell(BaseModel):
     @classmethod
     def from_dict_format(cls, data: Any) -> Any:
         """from_dict_format."""
-        if isinstance(data, Dict):
+        if isinstance(data, dict):
```

Comment:
This looked like a bug but apparently it's not:
```python
>>> from typing import Dict
>>> isinstance({}, Dict)
True
>>> isinstance({}, dict)
True
```
Replaced it to avoid confusion. Typehints shouldn't be used for instance checks